searchinrotatedsortedarray/search-in-rotated-sorted-array.js

```diff
@@ -0,0 +1,19 @@
+// https://leetcode.com/problems/search-in-rotated-sorted-array/
+function searchInRotatedSortedArray(nums,target){
+    const array = nums.concat(target)
+    const mySet = new Set()
```

Comment:
stosowanie const na set nie ma sensu bo set sie zmienia metodami na ktore const wplywu nie ma. to samo z tablica wyzej. NIe uzywamy const bo to bez sensu zazwyczaj :D